src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -27,14 +35,83 @@ const propTypes = {
 
     /** Whether to allow option focus or not */
     shouldDisableFocusOptions: PropTypes.bool,
+
+    /** The policy which the user has access to and which the report could be tied to */
+    policy: PropTypes.shape({
+        /** The ID of the policy */
+        id: PropTypes.string,
+        /** Name of the policy */
+        name: PropTypes.string,
+        /** Avatar of the policy */
+        avatar: PropTypes.string,
+    }),
+
+    /** All reports shared with the user */
+    reports: PropTypes.objectOf(reportPropTypes),
+
+    /** Array of report actions for this report */
+    reportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
+
+    /** Indicates which locale the user currently has selected */
+    preferredLocale: PropTypes.string,
+
+    /** List of users' personal details */
+    personalDetails: PropTypes.objectOf(participantPropTypes),
+
+    /** The transaction from the parent report action */
+    transactions: PropTypes.objectOf(
+        PropTypes.shape({
+            /** The ID of the transaction */
+            transactionID: PropTypes.string,
+        }),
+    ),
 };
 
 const defaultProps = {
     style: styles.flex1,
     shouldDisableFocusOptions: false,
+    reportActions: {},
+    reports: {},
+    policy: {},
+    preferredLocale: CONST.LOCALES.DEFAULT,
+    personalDetails: {},
+    transactions: {},
 };
 
-function LHNOptionsList({style, contentContainerStyles, data, onSelectRow, optionMode, shouldDisableFocusOptions}) {
+const keyExtractor = (item) => item;
+
+function LHNOptionsList({
+    style,
+    contentContainerStyles,
+    data,
+    onSelectRow,
+    optionMode,
+    shouldDisableFocusOptions,
+    reports,
+    reportActions,
+    policy,
+    preferredLocale,
+    personalDetails,
+    transactions,
+}) {
+    const itemPersonalDetails = useMemo(
+        () =>
+            _.reduce(
+                personalDetails,
+                (finalPersonalDetails, personalData, accountID) => {
+                    // It's OK to do param-reassignment in _.reduce() because we absolutely know the starting state of finalPersonalDetails
+                    // eslint-disable-next-line no-param-reassign
+                    finalPersonalDetails[accountID] = {
+                        ...personalData,
+                        accountID: Number(accountID),
```

Comment:
Why isn't accountID a number already? Why aren't we storing it in the DB as a number? 

It doesn't feel like it's the job of this list component to make sure the accountID is a number.

Reply:
Good catch, previously it was living in `OptionRowLHNData` component and it was copied here, i think it doesn't make sense to parse this to number because `accountID` is already a number in `personalData`. Removed this line.